src/julia.h

```diff
@@ -2274,6 +2274,15 @@ typedef struct _jl_handler_t {
 
 #define JL_RNG_SIZE 5 // xoshiro 4 + splitmix 1
 
+// delta_timestamp values are nanoseconds since the boot-time of the julia runtime system.
```

Comment:
That's an overflow every 4 seconds (vs uint64_t which was an overflow every 5 centuries). I think you need to reduce precision somewhat?

Reply:
Yeah, i realized the same. I think uint32 is going to be just too small to be useful for measuring time. Discussing [on slack](https://julialang.slack.com/archives/C02H0E8S2NA/p1732133591721809?thread_ts=1732132591.394879&cid=C02H0E8S2NA).

Reply:
reverted